src/styles/blocks/analog-clock.scss

```diff
@@ -0,0 +1,73 @@
+.analog-clock {
+  position: relative;
+  width: 500px;
+  height: 500px;
+
+  background-color: rgba(148, 144, 187, 0.778);
+  border-radius: 50%;
+
+  &__clock-hand {
+    position: absolute;
+    left: 50%;
+    bottom: 50%;
+
+    border-radius: 50% 50% 0 0;
+
+    animation-name: ticking;
+    animation-timing-function: steps(60);
```

Comment:
minutes and hours hands usually don't tick once in a hour/minute, their movement is smooth, so don't use `steps` for these hands